### DISCUSSION TITLE
UNISWAP V3

### DISCUSSION THREAD

**Discussion by 0xkumar** [Cyfrin/foundry-full-course-cu#1470] [General]
Hi guys,
I want to fetch the prices of token pairs from the uniswap v3 pool. But I have no idea how can I do that Can anyone help me  to get the prices of tokens from the uniswap v3 pools.Thank you

**Comment by EngrPips** [comment]
Hello @0xkumar , I would advice you go ahead and read the uniswapV3 documentation there should be every details on how to fetch price from their pool in their documentation. below is the link that would lead you to the page on uniswapV3 docs that discuss about getting the price of a token from it's pool 

https://docs.uniswap.org/sdk/v3/guides/swaps/quoting

**Comment by Berimor11** [comment]
great job

**Comment by vsland1** [comment]
No way, man!

**Reply by EngrPips** [reply]
hmmm 🤔

**Comment by AlexeyMasnoy** [comment]
Hi guys,

**Reply by EngrPips** [reply]
Hello, Friend. How is it going?

**Comment by Leonardoloot** [comment]
nice work!

**Reply by EngrPips** [reply]
Indeed
